### DISCUSSION TITLE
Does release or tag badges support filtering by a prefix?

### DISCUSSION THREAD

**Discussion by xingao267** [badges/shields#6253] [General] [answered]
For example, our repo has two sets of releases, foo-v1.0.0 and bar-v2.0.0 and each is released independently. We would like to have two separate badges to track the latest release of each.

**Comment by calebcartwright** [comment, marked as answer]
I assume you are referring to the GitHub latest Release and Tag Badges?

If so then no not really.  There is an optional asset-name param you can use on the GitHub Release Downloads badge that supports a use case in a similar vein, so I could see such a filtering option being useful on the latest version badges too (we publish two different sets of tags ourselves in this very repo).

More generally, a good self-service way to check the available parameters on a badge is to use the Shields.io site. From there you can locate your target badge (either via searching using the box at the top of the badge or clicking on categories and navigating), and then click to launch the modal badge builder window which shows any custom params supported by that badge, along with a few of the [default customization params that all badges support](https://shields.io/#styles):

![image](https://user-images.githubusercontent.com/13042488/110687391-0f871f80-81a6-11eb-812d-e373d8c80553.png)

**Reply by xingao267** [reply]
any chance to support filtering for GitHub latest release or tag badge?

**Reply by calebcartwright** [reply]
I agree such a feature could be useful as I mentioned in my previous comment. If this is something you'd like to see then please feel free to open a feature request (https://github.com/badges/shields/issues/new/choose) or even better, consider submitting a pull request!

**Reply by chris48s** [reply]
Hello. We have just shipped a new feature which allows you to pass an arbitrary filter to the github tag and release badges. For example

The `filter` param can be used to apply a filter to the project's tag or release names before selecting the latest from the list. Two constructs are available: `*` is a wildcard matching zero or more characters, and if the pattern starts with a `!`, the whole pattern is negated.

e.g:
https://img.shields.io/github/v/tag/badges/shields
https://img.shields.io/github/v/tag/badges/shields?filter=%21server-%2A

This feature should be powerful and flexible enough to enable this use-case (and a bunch of others).
